lib/ansible/modules/windows/win_secedit.ps1

```diff
@@ -0,0 +1,179 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2016, Red Hat Inc
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+########
+
+
+Set-StrictMode -Version Latest
+
+function Get-IniFile {
+    param (
+        [parameter(mandatory=$true, position=0, valuefrompipelinebypropertyname=$true, valuefrompipeline=$true)][string]$FilePath
+    )
+
+        $ini = New-Object System.Collections.Specialized.OrderedDictionary
+        $currentSection = New-Object System.Collections.Specialized.OrderedDictionary
+        $curSectionName = "default"
+
+        switch -regex (gc $FilePath)
+        {
+            "^\[(?<Section>.*)\]"
+            {
+                        $ini.Add($curSectionName, $currentSection)
+
+                        $curSectionName = $Matches['Section'].trim()
+                        $currentSection = New-Object System.Collections.Specialized.OrderedDictionary
+            }
+                "(?<Key>.*)\=(?<Value>.*)"
+                {
+                        # add to current section Hash Set
+                        $currentSection.Add($Matches['Key'].trim(), $Matches['Value'].trim())
+                }
+                "^$"
+                {
+                        # ignore blank line
+                }
+                default
+                {
+                        throw "Unidentified: $_"  # should not happen
+                }
+        }
+        if ($ini.Keys -notcontains $curSectionName) { $ini.Add($curSectionName, $currentSection) }
+
+        return $ini
+}
+
+function Out-IniFile{
+    param (
+        [parameter(mandatory=$true, position=0, valuefrompipelinebypropertyname=$true, valuefrompipeline=$true)][System.Collections.Specialized.OrderedDictionary]$ini,
```

Comment:
Probably want more input from the other Windows guys but I don't think we want to do pipelining in the modules if we can avoid it. I believe this is so we keep a similar standard between the python and powershell ones. @jhawkesworth @dagwieers @nitzmahone?